client/devdocs/sidebar.jsx

```diff
@@ -104,6 +104,13 @@ export default class DevdocsSidebar extends React.PureComponent {
 							link="/devdocs/blocks"
 							selected={ this.isItemSelected( '/devdocs/blocks', false ) }
 						/>
+						<SidebarItem
```

Comment:
I had the same doubt. My main concern is that there are some things we cannot hide even if the feature flag is not enabled:

* Gutenberg components in the [Components Glossary](https://github.com/Automattic/wp-calypso/blob/20a8b9491450868803a8c27974e46820ed7a2410/docs/components.md)
* Gutenberg components docs in the search: 
![screen shot 2018-07-26 at 20 33 55](https://user-images.githubusercontent.com/1233880/43281162-46bbddf0-9113-11e8-9204-2f43b72b51d8.png)

Because of that and because all the Gutenberg components are actually ready to be used in Calypso (we just need to add more docs explaining how to use the rest of them), I thought that was ok to display this publicly. 

But I don't have a strong opinion about this, honestly. If you think that having the readme files available is not a big deal, I can put this under the same feature flag.